### DISCUSSION TITLE
[Android] app as overlay over other apps (transparent background?)

### DISCUSSION THREAD

**Discussion by Rickard87** [dotnet/maui#23814] [Q&A]
[Android]
I wonder if it is possible to make an app background transparent with visible buttons (and) have that app be always on top, so that it is possible to open another app.
Why?
I have an app that sends commands to a map app using http requests to loopback address. But instead of switching between the 2 apps I'd like for the buttons to be available on top of the map app. I do not own the map app and cannot make any changes to it.
1. Can this be done in native Android Studio?
2. Can this be done in MAUI?

I have on my phone a blue light filter that works regardless of what app is opened so that's a kind of transparency with color. It even has an icon down at the right side on the navigation bar to activate or deactivate it.

The phones I will install it on are my own, so I am not interested in setting permissions for others by uploading it to Play Store for example. Whatever can be modified on my own phone is what goes.
I'd appreciate some help with if it's possible or what I can do?

**Comment by CaseAlexander** [comment]
Did you find a solution for this?  I'm in a similar situation - enterprise app for private devices (not public of on Play Store).  We need to overlay on top of videos (like from youtube or netflix)

**Comment by sjordanGSS** [comment]
Have you tried setting ```android:theme="@android:style/Theme.Translucent.NoTitleBar.Fullscreen"``` on your ```activity``` in ```AndroidManifest.xml```?
